src/components/RichTextEditor/plugins/linkChooserPlugin/transforms/getAndUpsertLink.ts

```diff
@@ -9,17 +9,29 @@ export const getAndUpsertLink = async <T = {}>(
     editor: PlateEditor<T>,
     getChosenLink?: LinkChooserPlugin['getChosenLink'],
 ) => {
-    const type = ELEMENT_LINK_CHOOSER;
+    const types = [ELEMENT_LINK, ELEMENT_LINK_CHOOSER];
```

Comment:
I just wondered if it wouldn't be better if we stick to `ELEMENT_LINK` instead of introducing a new element type and guaranteeing compatibility to the previous version